renderdoc/api/replay/renderdoc_replay.h

```diff
@@ -1021,6 +1021,17 @@ bucket when the pixel values are divided between ``minval`` and ``maxval``.
   virtual ShaderDebugTrace *DebugThread(const rdcfixedarray<uint32_t, 3> &groupid,
                                         const rdcfixedarray<uint32_t, 3> &threadid) = 0;
 
+  DOCUMENT(R"(Retrieve a debugging trace from running a mesh shader.
+
+:param Tuple[int,int,int] groupid: A list containing the 3D workgroup index.
+:param Tuple[int,int,int] threadid: A list containing the 3D thread index within the workgroup.
+:return: The resulting trace resulting from debugging. Destroy with
+  :meth:`FreeTrace`.
+:rtype: ShaderDebugTrace
+)");
+  virtual ShaderDebugTrace *DebugMeshletThread(const rdcfixedarray<uint32_t, 3> &groupid,
```

Comment:
I've pushed a fix to the python script which verifies docstrings and was causing the CI failure. It previously had an exception for DebugThread but it wasn't too hard to handle mapping `Tuple[int,int,int]` to `rdcfixedarray<uint32_t, 3>`.

Nothing for you to do, it should just work if you rebase onto latest v1.x.